frontend-ssr/pages/html/index.tsx

```diff
@@ -37,7 +41,10 @@ export default class Html extends React.Component {
           <Link href="/">
             Back to home
           </Link>
-          <MatchCommandClassComponent commands={expressionToMatch} />
+          <MatchCommandClassComponent
+            commands={expressionToMatch.map((item) => item.command)}
```

Comment:
Sorry, i don't think I was explicit enough. Let's modify the props as well, instead of 2 let's have just one, the array of objects.

Same logic as above, better for them to pe "packed" together